python/download_tile.py

```diff
@@ -0,0 +1,77 @@
+from looker_sdk import client, models
+import time
+import sys
+
+sdk = client.setup("../looker.ini")
+
+def get_dashboard(title: str):
+  """Get a dashboard by title"""
+  dashboard = next(iter(sdk.search_dashboards(title=title)), None)
+  if not dashboard:
+    print(f"dashboard {title} was not found")
+  return dashboard
+
+def get_tile(dash: models.Dashboard, title: str):
+  """Find a dashboard tile by title"""
+  title = title.lower()
+  found = None
+  assert dash.dashboard_elements
+  for tile in dash.dashboard_elements:
+    assert tile.title
+    if tile.title.lower() == title:
+      found = tile
+  if not tile:
+    print(f"tile {title} of dashboard {dash.title} was not found")
+  return tile
+
+def download_tile(tile: models.DashboardElement, format:str = 'png'):
+  """Download the dashboard tile in the specified format"""
+  if not tile.query_id:
+    print(f"tile {tile.title} has no query_id")
+    return None
+  print(tile.query_id)
```

Comment:
debug statement?

Reply:
yeah, left that in on accident

Reply:
```suggestion
```

Reply:
oh fancy, I didn't know you could delete a line with a suggestion! ah, it's just an empty markdown code block! genius :-)